.gitignore

```diff
@@ -10,4 +10,5 @@ __pycache__
 1.cellprofiler_ic/Corrected_Plate_2
 
 # ignore log files
-1.cellprofiler_ic/logs
\ No newline at end of file
+1.cellprofiler_ic/logs
+2.cellprofiler_analysis/logs
```

Comment:
Would a wildcard be helpful here to avoid any directory with logs (in a single line)?

Reply:
I agree it would since logs are used in multiple modules/directories.